<launch>

    <!-- Configurable arguments -->
    <arg name="robot_name" default="sam"/>

	<!-- topics, note the root! -->
	<arg name="emergency_topic" default="/$(arg robot_name)/abort" />
    <arg name="vbs_cmd_topic" default="/$(arg robot_name)/core/vbs_cmd" />
    <arg name="rpm_cmd_topic" default="/$(arg robot_name)/core/rpm_cmd" />
	<arg name="lcg_pid_enable_topic" default="/$(arg robot_name)/ctrl/lcg/pid_enable" />
    <arg name="vbs_pid_enable_topic" default="/$(arg robot_name)/ctrl/vbs/pid_enable" />
    <arg name="tcg_pid_enable_topic" default="/$(arg robot_name)/ctrl/tcg/pid_enable" />
	<arg name="yaw_pid_enable_topic" default="/$(arg robot_name)/ctrl/dynamic_heading/pid_enable" />
    <arg name="depth_pid_enable_topic" default="/$(arg robot_name)/ctrl/dynamic_depth/pid_enable" />
    <arg name="vel_pid_enable_topic" default="/$(arg robot_name)/ctrl/dynamic_velocity/pid_enable" />
    
    <node name="emergency_surface_action" pkg="auv_simple_motion_planner" type="emergency_surface_action.py" output="screen" ns="$(arg robot_name)/ctrl" pass_all_args="true" />

</launch>
